src/pages/add-goal/add-goal.ts

```diff
@@ -4,34 +4,26 @@ import { NavController } from 'ionic-angular';
 
 import { Storage } from '@ionic/storage';
 
-import { User } from '../../providers/user';
+import { CreateGoalSettingsPage } from '../create-goal-settings/create-goal-settings';
+
+import { GoalWizardPage } from '../goal-wizard/goal-wizard';
 
 @Component({
 	selector: 'page-add-goal',
 	templateUrl: 'add-goal.html'
 })
-export class AddGoalPage {
-	access_token: string;
 
-	constructor(public navCtrl: NavController, public storage: Storage, public user: User) {}
-
-	onSubmit(formData) {
-		let decade = 60 * 60 * 24 * 365 * 10;
-		let d = new Date();
-		let t = Math.floor(d.getTime() / 1000);
+export class AddGoalPage {
 
-		var goaldate = t + decade;
+	constructor(public navCtrl: NavController, public storage: Storage) {}
 
-		let goal = {
-			slug: formData.slug,
-			title: formData.goaltitle,
-			goal_type: formData.type,
-			goaldate: goaldate,			
-			rate: formData.rate,
-			gunit: formData.gunits,
-			runit: formData.runits,
-		}
-		this.user.addGoal(goal);
-	}
+	goToCreateManualGoal() {
+	  this.storage.set('isManualGoal', true);
```

Comment:
Unnecessarily to set something in storage, when navctrl can pass parameters to other pages.